apps/editors/forms.py

```diff
@@ -250,11 +247,7 @@ def validate(self, value):
         pass
 
 
-class ReviewAddonForm(happyforms.Form):
-    addon_files = AddonFilesMultipleChoiceField(
-        required=False,
-        queryset=File.objects.none(), label=_lazy(u'Files:'),
-        widget=forms.CheckboxSelectMultiple())
+class ReviewForm(happyforms.Form):
```

Comment:
We don't want the file checkboxes anymore, so we don't need a different form for reviewing a new Addon (with all its files) or an update.